### PR TITLE
RWA-587: Add new postgres 11 database through terraform configuration

### PR DESCRIPTION
**Pull request by alexbaeza**
### Change description ###

Add new postgres 11 database through terraform configuration

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: infrastructure/variables.tf [hunk 1]
```diff
@@ -19,3 +19,13 @@ variable "common_tags" {
 variable "appinsights_instrumentation_key" {
   default = ""
 }
+
+
+variable "postgresql_database_name" {
+  default = "wa_task_management_api"
```
Comment (raghera) [inline infrastructure/variables.tf:25]: I know this is just the default but wondered if we want to call it something else?  `cft-task-db` ?  To match with the domain language?

Comment (alexbaeza) [reply infrastructure/variables.tf:25]: sure will apply the change

File: infrastructure/variables.tf [hunk 2]
```diff
@@ -19,3 +19,15 @@ variable "common_tags" {
 variable "appinsights_instrumentation_key" {
   default = ""
 }
+
+variable "postgres_db_component_name" {
```
Comment (iguannago) [inline infrastructure/variables.tf:23]: do we need this? the wa-workflow-api does not have it, though

**Comment by iguannago** [comment]
Shall we also add the DB env vars to both the bootstrap and Charts?

File: infrastructure/variables.tf (hunk 2 above)
Comment (alexbaeza) [reply infrastructure/variables.tf:23]: as per suggestion from @raghera we want to name this database host and db differently so I saved it as a variable so it's easier to control.

**Review by iguannago — APPROVED** [review]

**Comment by alexbaeza** [comment]
> Shall we also add the DB env vars to both the bootstrap and Charts?

Hey @iguannago This is done as part of the second PR:
https://github.com/hmcts/wa-task-management-api/pull/266

As you can see the second PR fails as I need to create the infrastructure and let terraform add those secrets into the vault however terraform only runs on master, so I have split the PRs into two.

**Pull request merged**